packages/legacy/core/App/navigators/ConnectStack.tsx

```diff
@@ -17,17 +17,22 @@ const ConnectStack: React.FC = () => {
   const defaultStackOptions = createDefaultStackOptions(theme)
   const { scan } = useConfiguration()
   const { t } = useTranslation()
-  const [store] = useStore()
+
   return (
     <Stack.Navigator
-      // below is part of the temporary gating of the new scan screen tabs feature
-      screenOptions={{ ...defaultStackOptions, headerShown: store.preferences.useConnectionInviterCapability }}
+      screenOptions={{
+        ...defaultStackOptions,
+      }}
     >
       <Stack.Screen name={Screens.Scan} component={scan} />
+      <Stack.Screen name={Screens.ScanHelp} component={ScanHelp} />
```

Comment:
Did you mean to make this component come from config so it can be overridden? I'm assuming that's why the `whereToUseWalletUrl` in `ScanHelp.tsx` was to a throwaway url

Reply:
Oh ok. I thought at least the URL would have to be different for each wallet project or beyond that even the general content of the ScanHelp screen would have to be different as well

Reply:
Confirmed. Translation work as expected although I am missing them in BC Wallet. Will fix here but that should not impact this PR as it is only for Bifold.